lab_4_retrieval_w_clustering/main.py

```diff
@@ -27,6 +29,10 @@ def get_paragraphs(text: str) -> list[str]:
         list[str]: Paragraphs from document.
     """
 
+    if not text:
+        raise ValueError
+    return text.split("\n")
```

Comment:
there can be blank strings after this operation, so result after parsing this input
```
hello, my friend!\n
\n
hello!
```
will be ['hello, my friend!','hello!']

Reply:
Done